src/IntercomCompanies.php

```diff
@@ -33,6 +33,37 @@ public function update($options)
         return $this->create($options);
     }
 
+    /**
+     * Attaches a Contact to a Company.
+     *
+     * @see    https://developers.intercom.io/reference#attach-contact-to-company
+     * @param  string $contactId
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function attachContact($contactId, $options)
+    {
+        $path = $this->companyAssociatePath($contactId);
+        return $this->client->post($path, $options);
+    }
+
+    /**
+     * Detaches a Contact from a Company.
+     *
+     * @see    https://developers.intercom.io/reference#detach-contact-from-company
+     * @param  string $contactId
+     * @param  string $companyId
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function detachContact($contactId, $companyId, $options = [])
+    {
+        $path = $this->companyAssociatePath($contactId);
+        return $this->client->delete($path . '/' . $companyId, $options);
```

Comment:
Would be great to create a method to store the complexity of this path, ie `$this->companyDetachPath($contactId, $companyId);`